internal/core/software_renderer.rs

```diff
@@ -594,7 +594,10 @@ struct SceneTexture<'a> {
     /// bytes between two lines in the source
     stride: u16,
     source_size: PhysicalSize,
+    /// Color to colorize. When not transparent, consider that the image is an alpha map and always use that color.
+    /// The alpha of this color is ignored. (it is supposed to be mixed in `Self::alpha`)
     color: Color,
+    /// Global alpha
```

Comment:
I'm curious, why is this called "global", when it incorporates for example the local alpha from the text color? I'd call it just "alpha".

Reply:
Yeah, this is the alpha, the comment is indeed misleading

```suggestion
```